### PR TITLE
Development

### PR DESCRIPTION
**Pull request by moudio**

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @MouhaDiouf,
You did a really good job building this web page. It only needs some minor changes to be in accord with the newest Microverse guidelines.
## Part 1 : _You need to work on the mobile view, it a little bit off compare to the original website. checkout the screenshot below from your web page;_ 
image

check out the PC view too;
image

## Part 2: Here is the checklist for your project.
- To consider the project 'complete' it must satisfy the following points:
## General
- [ ] The project has a descriptive README.md
## Mobile \[Screen `width` up to `480px`\]
## Navigation Bar
* [ ] It is wrapped around a pair of `<nav>` HTML tags
* [ ] It has the Newsweek logo on the leftmost part of the navbar
* [ ] It has the following icons in the listed order on the rightmost part of the navbar
  * [ ] Sign in
  * [ ] Subscribe
  * [ ] Hamburger menu icon (three horizontal bars)
## Others
* [ ] Top Story, More Stories, Featured Stories are each one a separated section. Use 3 sections for each one of them.
* [ ] Each one of the More Stories must be an article.
* [ ] Opinion container must be a section tag.
* [ ] Each one of the 8 items in the opinion section must inside an article tag.
* [ ] The Latest News section must be a section tag. Each one must be an article tag and be inside one section tag for the whole section.
* [ ] The 'In the magazine' section must be a section tag. Each one must be an article tag and be inside one section tag for the whole section.
* [ ] The 'Editor's pick' must be a section tag. Each one must be an article tag and be inside one section tag for the whole section.
* [ ] The 'Featured Slideshow' must be a section tag. Each one must be an article tag and be inside one section tag for the whole section.

## Important
**_Check out this [link](https://www.themelocation.com/best-html5-practices) for the right way to structure your HTML tags._**

**Kindly work on the issues from Part 1 & 2 and submit another code review form.**

File: index.html [hunk 1]
```diff
@@ -41,19 +43,35 @@
...
     </nav>
 </header>
 <main>
-    <div class="container">
+    <div class="container-fluid">
 <div class="row">
-    <div class="col-md-3 featured-stories">
+    <section class="col-lg-3 col-md-5 featured-stories">
         <h2 class="title">FEATURED STORIES</h2>
         <div class="story">
```
Comment (Torres-ssf) [inline index.html]: Should be an `article` tag.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by Torres-ssf** [comment]
### Awesome @MouhaDiouf. You did a great job on this page, be proud of it. 👍
It's missing only a few things. I trust you to make these changes and merge for master afterward. So no need to submit for code review again. They are:

- [ ] Every one of the opinion containers should be inside an `article` tag. The one with `class="small-opinion"`

- [ ] Every story container should be inside an `article` tag as well. Please change all 4 `div class="story"` to an `article` tag. And the 2 `class="story sponsored-insights"` have to be in the same approach. In this project, every article must be inside an `article` tag. The reason for doing this is because the HTML5 tags add real value to your code. They all have a semantic meaning. Check out this article to read more about the importance of semantic HTML https://www.lifewire.com/why-use-semantic-html-3468271.

Sorry to ask you to make changes again @MouhaDiouf but these are Microverse guidelines. After making the changes you can merge to master and submit the form. Good luck in the next projects

**Comment by moudio** [comment]
Thanks, I'll apply the changes

Le ven. 28 juin 2019 à 12:32, Sérgio Torres <notifications@github.com> a
écrit :

**Pull request merged**